modal/sandbox.py

```diff
@@ -743,8 +754,9 @@ async def exec(
         # Control line-buffered output.
         # -1 means unbuffered, 1 means line-buffered (only available if `text=True`).
         bufsize: Literal[-1, 1] = -1,
-        # Internal option to set terminal size and metadata
-        _pty_info: Optional[api_pb2.PTYInfo] = None,
+        pty: bool = False,  # Enable a PTY for the command
```

Comment:
Since we do not have `*` to make these keyword only, I do not think we can change the order of these parameters without the possibility of breaking code.

Reply:
Yes but (as @mwaskom [pointed out to me](https://github.com/modal-labs/modal-client/pull/3548#discussion_r2344468809) just the other day), the `*args` variadic args effectively forces the rest to be kwargs!